microsoft/testsuites/network/networksettings.py

```diff
@@ -96,6 +118,13 @@ def validate_device_channels_change(self, node: Node) -> None:
             channels = interface_channels_info.current_channels
             max_channels = interface_channels_info.max_channels
 
+            if max_channels <= 1:
```

Comment:
Is there any exception should be raised, or the test should be skipped? How about all of them are 1?

Reply:
Yeah, we cannot attempt to change it to < 1 and if max is just 1 then there is no value that we can test it against.

Reply:
I'm not sure if other things can be validated in this case too. If there is no, it should be skipped, because it's not testable. If there are other things to validate, we may can split it into two cases. One is skipped, and another one can be validated.

Reply:
Its like one device might have max channels as 1 but other devices (if present) might have different value.